arangod/Cache/FrequencyBuffer.h

```diff
@@ -122,18 +109,21 @@ class FrequencyBuffer {
     for (auto f : frequencies) {
       data.emplace_back(std::pair<T, std::size_t>(f.first, f.second));
```

Comment:
size_t incorrect here
```suggestion
      data.emplace_back(f.first, f.second);
```

Reply:
Changed.